pkg/controller/cassandradatacenter/controller/jvm.go

```diff
@@ -0,0 +1,113 @@
+package controller
+
+import (
+	cassandraoperatorv1alpha1 "github.com/instaclustr/cassandra-operator/pkg/apis/cassandraoperator/v1alpha1"
+	"strconv"
+	"strings"
+)
+
+func cassandraJVMOptions(cdc *cassandraoperatorv1alpha1.CassandraDataCenter) (*string, error) {
```

Comment:
We don't need to check for errors -- we are writing to an in-memory buffer, not to a file. The Go docs even ignore the result of `Fprintf` in the case of it writing to a `strings.Builder`.

Reply:
thats good news, we can further get rid of that then, I am not sure how it will play with some go code quality tools we can apply here in the future, they would maybe mark them as false positives that we have not checked them and code quality rating could suffer